main/vala/APKBUILD

```diff
@@ -1,31 +1,50 @@
-# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Contributor: Natanael Copa <ncopa@alpinelinux.org>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=vala
-pkgver=0.44.3
-pkgrel=1
+pkgver=0.44.5
+pkgrel=0
 pkgdesc="Compiler for the GObject type system"
 url="https://wiki.gnome.org/Projects/Vala"
 arch="all"
-options="!check" # Test requires dbus-launch.
 license="LGPL-2.0-or-later"
 subpackages="$pkgname-doc"
 depends="glib-dev"
-makedepends="libxslt-dev bash flex bison gobject-introspection-dev graphviz-dev"
+makedepends="libxslt-dev bash flex bison gobject-introspection-dev graphviz-dev
+	automake autoconf libtool"
+checkdepends="dbus-x11"
 source="https://download.gnome.org/sources/vala/${pkgver%.*}/vala-$pkgver.tar.xz
 	version.patch"
 
+prepare() {
+	default_prepare
+
+	# https://gitlab.gnome.org/GNOME/vala/issues/809
+	sed '/array-uint8-uchar-compat.vala/d' -i tests/Makefile.am
+
+	autoreconf --install --force
+}
+
 build() {
 	cd "$builddir"
+	export CFLAGS="-O2 -pipe"
```

Comment:
I wonder why this is needed?

Reply:
Oh my, good catch. Tried this to debug the one failing test. Will remove :)

Reply:
Fixed.